frontend/src/app/App.tsx

```diff
@@ -40,18 +42,40 @@ datadogLogs.init({
   sampleRate: 100,
 })
 
-export const App = (): JSX.Element => (
-  <HelmetProvider>
-    <QueryClientProvider client={queryClient}>
-      <ReactQueryDevtools initialIsOpen={false} />
-      <AppHelmet />
-      <BrowserRouter>
-        <ChakraProvider theme={theme} resetCSS>
-          <AuthProvider>
-            <AppRouter />
-          </AuthProvider>
-        </ChakraProvider>
-      </BrowserRouter>
-    </QueryClientProvider>
-  </HelmetProvider>
-)
+export const App = (): JSX.Element => {
+  const isDev = process.env.NODE_ENV === 'development'
+
+  return (
+    <>
+      {isDev && (
+        <Inspector
+          // props see docs:
+          // https://github.com/zthxxx/react-dev-inspector#inspector-component-props
+          keys={['control', 'shift', 'command', 'c']}
```

Comment:
suggestion: use key combi without command key, in case someone is non-macos user